kiwi/bootloader/config/grub2.py

```diff
@@ -461,7 +461,25 @@ def __get_module_path(self, format_name, lookup_path=None):
             ]
         )
 
+    def __fixup_legacy_grub_location(self):
+        legacy_grub_theme_dir = self.root_dir + '/boot/grub/themes/'
+        legacy_font = self.root_dir + '/boot/grub/unicode.pf2'
+        grub_dir = self.root_dir + \
+            '/boot/' + self.boot_directory_name + '/'
+        if os.path.exists(legacy_grub_theme_dir):
+            # found grub2 theme directory in legacy grub directory
+            Path.create(grub_dir)
+            Command.run(
+                ['cp', '-a', legacy_grub_theme_dir, grub_dir]
+            )
```

Comment:
I would avoid `Command.run` at all costs. Especially if there is [`shutil`](https://docs.python.org/3/library/shutil.html) for this.